reflex/utils/prerequisites.py

```diff
@@ -398,7 +427,9 @@ def check_initialized(frontend: bool = True):
         Exit: If the app is not initialized.
     """
     has_config = os.path.exists(constants.CONFIG_FILE)
-    has_reflex_dir = not frontend or IS_WINDOWS or os.path.exists(constants.REFLEX_DIR)
+    has_reflex_dir = (
+        not frontend or constants.IS_WINDOWS or os.path.exists(constants.REFLEX_DIR)
```

Comment:
```suggestion
        not frontend or os.path.exists(constants.REFLEX_DIR)
```

all platforms should have a `REFLEX_DIR` now

Reply:
ah yes. Good spot